### ISSUE TITLE
Let the 'Bot ASK if it Should Create a Matrix Community a.k.a. Group

### ISSUE THREAD

**Issue by MilkManzJourDaddy** [turt2live/matrix-hashtag-bot#3]
Interesting but spammy.  I suppose if room members do not mind the spam-tastic aspect; if the when triggered, the 'bot ***asked*** if it should create a group/community, that would mean less database clutter and bedlam.  

On platforms like Twitter™, hashtags have associated messages, instead of just "virtue signalling", or flair, or something.  Matrix already has flair with Communities/Groups, but they have Admin' users.   IDK.  

But if the 'bot ***asked*** if it should create a group, also prompting the help info' with triggers and the "join" bit; that might be more functional and less cluttering groups/communities where someone was being silly, or regex did not catch an unintended item.

**Comment by MilkManzJourDaddy** [comment]
Yes, worry about downvoting this and "putting [my] mind to rest", rather than: [Groups code over federation hammers servers causing 429s and marking the server down](https://github.com/matrix-org/synapse/issues/3549).  "Brilliant".

**Comment by Half-Shot** [comment]
I mean, maybe you should be fixing the problem with synapse rather than complaining about one bot that uses it.
